src/main/resources/templates/start.html

```diff
@@ -16,8 +16,8 @@
 
     <div class="container m-3 ">
         <div class="builds-header row">
-            <h3> My Builds </h3>
-            <button type="button" class="btn btn-outline-secondary">Add New Build</button>
+            <h3> My Builds &nbsp  </h3>
```

Comment:
Added &nbsp tag so that there is space between the text in h3 tag and button tag. This will look good for future.

Reply:
Since h3 is a line breaking tag, you won't see the whitespace added here.
Also, consider using CSS for margin and padding instead of whitespace nbsp and br tags.  CSS is more fluid across multiple resolutions.